### PR TITLE
fix: table scrolling issues

### PR DESCRIPTION
**Pull request by jake-bassett**
## Description
Fixes for some table scrolling issues

### CODE REVIEW

File: projects/components/src/table/table.component.scss [hunk 1]
```diff
@@ -39,7 +38,7 @@
 .data-row {
   display: flex;
   flex-direction: row;
-  height: 44px;
+  min-height: 44px;
```
Comment (anandtiwary) [inline projects/components/src/table/table.component.scss:41]: rows can possibly expand with this change

Comment (jake-bassett) [reply projects/components/src/table/table.component.scss:43]: Fixed.

File: projects/components/src/table/table.component.scss [hunk 2]
```diff
@@ -106,7 +107,7 @@
 }
 
 .state-watcher {
-  height: 100%;
+  height: calc(100% - #{$paginator-height});
```
Comment (anandtiwary) [inline projects/components/src/table/table.component.scss:110]: curious. what does #{} do?

File: projects/components/src/table/table.component.scss (hunk 1 above)
Comment (anandtiwary) [reply projects/components/src/table/table.component.scss:43]: hmm.. why do we need both height and min-height?

File: projects/components/src/table/table.component.scss (hunk 2 above)
Comment (jake-bassett) [reply projects/components/src/table/table.component.scss:110]: [Interpolation](https://sass-lang.com/documentation/interpolation). Its so we can use the SCSS variable in the CSS function. It works like html templates in typescript.

File: projects/components/src/table/table.component.scss (hunk 1 above)
Comment (jake-bassett) [reply projects/components/src/table/table.component.scss:43]: The new flex layout was trying to shrink the row height, so we needed the min-height so flex doesn't override the height. The height was just to address your valid comment about the row expanding.

**Review by aaron-steinfeld — COMMENTED** [review]
lgtm (except the scss, didn't review that)

**Review by anandtiwary — APPROVED** [review]

**Pull request merged**